src/app/icd/server/tests/TestICDManager.cpp

```diff
@@ -16,6 +16,7 @@
  *    limitations under the License.
  */
 
+#include "app-common/zap-generated/cluster-enums.h"
```

Comment:
use `<>` for includes to be consistent. Move them after the framework include

Reply:
Done